keyboards/fallacy/keymaps/via/config.h

```diff
@@ -0,0 +1,21 @@
+/* Copyright 2020 B. Fletcher (toraifu) <typefast@kyaa.gg>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+/* ensure NKRO is enabled by default
+ */
+#define FORCE_NKRO
```

Comment:
This is probably not a good idea, as the board may not work with some old BIOSes with this defined. VIA has the `NK_TOGG` keycode available so users can control which interface the board sends keystrokes on themselves.